frontend/buildspec.yml

```
@@ -25,22 +25,27 @@ phases:
       - echo "📦 Target S3 path: $S3_PATH"
 
   post_build:
-    commands:
-      - echo "🚀 Deploying to $S3_PATH with proper cache headers..."
-      - aws s3 cp dist/index.html $S3_PATH/index.html \
-        --cache-control "no-store" \
-        --content-type "text/html" \
-        --metadata-directive REPLACE
-      - aws s3 sync dist $S3_PATH/ \
-        --exclude "index.html" \
-        --cache-control "public, max-age=31536000, immutable" \
-        --metadata-directive REPLACE
-      - aws s3 sync frontend/certificate-image s3://techcourse-project-2025/fit-toring/certificate-image \
-        --cache-control "public, max-age=31536000, immutable" \
-        --metadata-directive REPLACE
-      - aws s3 sync frontend/profile-image s3://techcourse-project-2025/fit-toring/profile-image \
-        --cache-control "public, max-age=31536000, immutable" \
-        --metadata-directive REPLACE
+  commands:
+    - echo "🚀 Deploying to $S3_PATH with proper cache headers..."
+    - aws s3 cp dist/index.html $S3_PATH/index.html \
+      --cache-control "no-store" \
+      --content-type "text/html" \
+      --metadata-directive REPLACE
+    - aws s3 sync dist $S3_PATH/ \
+      --exclude "index.html" \
+      --cache-control "public, max-age=31536000, immutable" \
+      --metadata-directive REPLACE
+    - aws s3 cp s3://techcourse-project-2025/fit-toring/certificate-image/ \
+      s3://techcourse-project-2025/fit-toring/certificate-image/ \
+      --recursive \
+      --metadata-directive REPLACE \
+      --cache-control "public, max-age=31536000, immutable"
+    - aws s3 cp s3://techcourse-project-2025/fit-toring/profile-image/ \
+      s3://techcourse-project-2025/fit-toring/profile-image/ \
+      --recursive \
+      --metadata-directive REPLACE \
+      --cache-control "public, max-age=31536000, immutable"
+
```

p4: 질문인데 혹시 탭이 한 번 더 들어가지 않아도 괜찮을까? 🤔

```suggestion
    commands:
      - echo "🚀 Deploying to $S3_PATH with proper cache headers..."
      - aws s3 cp dist/index.html $S3_PATH/index.html \
        --cache-control "no-store" \
        --content-type "text/html" \
        --metadata-directive REPLACE
      - aws s3 sync dist $S3_PATH/ \
        --exclude "index.html" \
        --cache-control "public, max-age=31536000, immutable" \
        --metadata-directive REPLACE
      - aws s3 cp s3://techcourse-project-2025/fit-toring/certificate-image/ \
        s3://techcourse-project-2025/fit-toring/certificate-image/ \
        --recursive \
        --metadata-directive REPLACE \
        --cache-control "public, max-age=31536000, immutable"
      - aws s3 cp s3://techcourse-project-2025/fit-toring/profile-image/ \
        s3://techcourse-project-2025/fit-toring/profile-image/ \
        --recursive \
        --metadata-directive REPLACE \
        --cache-control "public, max-age=31536000, immutable"

```

---

[fix: tab 추가](https://github.com/woowacourse-teams/2025-Fit-toring/pull/732/commits/4b0eb611968708dcc72364d34caa299d27714aef)
꼼꼼하게 봐줘서 감사합니다..